lib/View.php

```diff
@@ -70,7 +70,7 @@ private function _scriptTag($file, $attributes = '')
         $sri = array_key_exists($file, $this->_variables['SRI']) ?
             ' integrity="' . $this->_variables['SRI'][$file] . '"' : '';
         // if the file isn't versioned (ends in a digit), add our own version
-        $cacheBuster = ctype_digit(substr($file, -4, 1)) ?
+        $cacheBuster = (bool) preg_match('#[0-9]\.js$#', (string) $file) ?
```

Comment:
Don't know what # means here, seems no regex thing, but anyway likely does not matter.

Reply:
The first character in a regex pattern is used as delimiter, to end the regex or separate the search from replacement patterns. Often the slash (`/`) is used for that, but when dealing with paths or HTML-snippets that requires to escape any slashes in the pattern. There seems to be a convention to use hash (`#`) in those cases, while you could use any character or letter that is not found inside the pattern. `'a[0-9]\.js$a'` would work just as well, for example.

For this one use, there is no benefit to use hash over slash.

Reply:
Ah missed the delim lol